_authors/ychen.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Yan Chen
+short_name: Yan
+twitter: 
```

Comment:
Remove this line if not needed
```suggestion
```

Reply:
Thanks for the feedback. I committed the changes accordingly. Do I need to open a new pull request?